### DISCUSSION TITLE
Undefined symbols for architecture arm64

### DISCUSSION THREAD

**Discussion by adlersam** [neperfepx/neper#451] [Q&A]
Hi there,

I'm new to Neper and I'm trying to install it on my Apple silicon Mac. It's giving me some trouble though. Here's what it's spitting out when I run `make` in the terminal. I'm pretty sure I've installed all of the required dependencies.

> Consolidate compiler generated dependencies of target nlopt
[  5%] Built target nlopt
Consolidate compiler generated dependencies of target scotch
[  5%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/arch.c.o
[  5%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/arch_build.c.o
[  5%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/arch_build2.c.o
[  5%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/arch_cmplt.c.o
[  6%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/arch_cmpltw.c.o
[  6%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/arch_deco.c.o
[  6%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/arch_deco2.c.o
[  6%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/arch_dist.c.o
[  6%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/arch_hcub.c.o
[  6%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/arch_mesh.c.o
[  6%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/arch_sub.c.o
[  6%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/arch_tleaf.c.o
[  6%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/arch_torus.c.o
[  7%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/arch_vcmplt.c.o
[  7%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/arch_vhcub.c.o
[  7%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/bgraph.c.o
[  7%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/bgraph_bipart_bd.c.o
[  7%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/bgraph_bipart_df.c.o
[  7%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/bgraph_bipart_ex.c.o
[  7%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/bgraph_bipart_fm.c.o
[  7%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/bgraph_bipart_gg.c.o
[  8%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/bgraph_bipart_gp.c.o
[  8%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/bgraph_bipart_ml.c.o
[  8%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/bgraph_bipart_st.c.o
[  8%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/bgraph_bipart_zr.c.o
[  8%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/bgraph_check.c.o
[  8%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/bgraph_store.c.o
[  8%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/common.c.o
[  8%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/common_file.c.o
[  9%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/common_file_compress.c.o
[  9%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/common_file_decompress.c.o
[  9%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/common_integer.c.o
[  9%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/common_memory.c.o
[  9%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/common_string.c.o
[  9%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/common_stub.c.o
[  9%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/common_thread.c.o
[  9%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/fibo.c.o
[ 10%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/gain.c.o
[ 10%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/geom.c.o
[ 10%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph.c.o
[ 10%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_base.c.o
[ 10%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_band.c.o
[ 10%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_check.c.o
[ 10%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_clone.c.o
[ 10%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_coarsen.c.o
[ 10%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_diam.c.o
[ 11%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_dump.c.o
[ 11%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_ielo.c.o
[ 11%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_induce.c.o
[ 11%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_io.c.o
[ 11%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_io_chac.c.o
[ 11%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_io_habo.c.o
[ 11%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_io_mmkt.c.o
[ 11%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_io_scot.c.o
[ 12%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_list.c.o
[ 12%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/graph_match.c.o
[ 12%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hall_order_hd.c.o
[ 12%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hall_order_hf.c.o
[ 12%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hall_order_hx.c.o
[ 12%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hgraph.c.o
[ 12%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hgraph_check.c.o
[ 12%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hgraph_dump.c.o
[ 13%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hgraph_induce.c.o
[ 13%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hgraph_order_bl.c.o
[ 13%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hgraph_order_cc.c.o
[ 13%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hgraph_order_cp.c.o
[ 13%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hgraph_order_gp.c.o
[ 13%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hgraph_order_hd.c.o
[ 13%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hgraph_order_hf.c.o
[ 13%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hgraph_order_hx.c.o
[ 13%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hgraph_order_kp.c.o
[ 14%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hgraph_order_nd.c.o
[ 14%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hgraph_order_si.c.o
[ 14%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hgraph_order_st.c.o
[ 14%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hmesh.c.o
[ 14%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hmesh_check.c.o
[ 14%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hmesh_hgraph.c.o
[ 14%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hmesh_induce.c.o
[ 14%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hmesh_mesh.c.o
[ 15%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hmesh_order_bl.c.o
[ 15%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hmesh_order_cp.c.o
[ 15%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hmesh_order_gr.c.o
[ 15%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hmesh_order_gp.c.o
[ 15%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hmesh_order_hd.c.o
[ 15%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hmesh_order_hf.c.o
[ 15%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hmesh_order_hx.c.o
[ 15%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hmesh_order_nd.c.o
[ 16%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hmesh_order_si.c.o
[ 16%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/hmesh_order_st.c.o
[ 16%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/kgraph.c.o
[ 16%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/kgraph_band.c.o
[ 16%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/kgraph_check.c.o
[ 16%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/kgraph_map_bd.c.o
[ 16%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/kgraph_map_cp.c.o
[ 16%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/kgraph_map_df.c.o
[ 17%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/kgraph_map_ex.c.o
[ 17%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/kgraph_map_fm.c.o
[ 17%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/kgraph_map_ml.c.o
[ 17%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/kgraph_map_rb.c.o
[ 17%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/kgraph_map_rb_map.c.o
[ 17%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/kgraph_map_rb_part.c.o
[ 17%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/kgraph_map_st.c.o
[ 17%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/kgraph_store.c.o
[ 17%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_arch.c.o
[ 18%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_arch_f.c.o
[ 18%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_arch_build.c.o
[ 18%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_arch_build_f.c.o
[ 18%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_arch_dom.c.o
[ 18%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_arch_dom_f.c.o
[ 18%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_common_f.c.o
[ 18%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_geom.c.o
[ 18%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_geom_f.c.o
[ 19%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph.c.o
[ 19%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_f.c.o
[ 19%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_base.c.o
[ 19%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_base_f.c.o
[ 19%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_check.c.o
[ 19%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_check_f.c.o
[ 19%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_coarsen.c.o
[ 19%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_coarsen_f.c.o
[ 20%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_color.c.o
[ 20%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_color_f.c.o
[ 20%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_diam.c.o
[ 20%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_diam_f.c.o
[ 20%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_dump.c.o
[ 20%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_induce.c.o
[ 20%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_induce_f.c.o
[ 20%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_io_chac.c.o
[ 20%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_io_chac_f.c.o
[ 21%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_io_habo.c.o
[ 21%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_io_habo_f.c.o
[ 21%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_io_mmkt.c.o
[ 21%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_io_mmkt_f.c.o
[ 21%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_io_scot.c.o
[ 21%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_io_scot_f.c.o
[ 21%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_map.c.o
[ 21%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_map_f.c.o
[ 22%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_map_io.c.o
[ 22%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_map_io_f.c.o
[ 22%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_map_view.c.o
[ 22%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_map_view_f.c.o
[ 22%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_order.c.o
[ 22%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_order_f.c.o
[ 22%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_part_ovl.c.o
[ 22%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_graph_part_ovl_f.c.o
[ 23%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_mapping.c.o
[ 23%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_memory.c.o
[ 23%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_memory_f.c.o
[ 23%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_mesh.c.o
[ 23%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_mesh_f.c.o
[ 23%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_mesh_graph.c.o
[ 23%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_mesh_graph_f.c.o
[ 23%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_mesh_io_habo.c.o
[ 24%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_mesh_io_habo_f.c.o
[ 24%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_mesh_io_scot.c.o
[ 24%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_mesh_io_scot_f.c.o
[ 24%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_mesh_order.c.o
[ 24%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_mesh_order_f.c.o
[ 24%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_order.c.o
[ 24%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_parser.c.o
[ 24%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_parser_f.c.o
[ 24%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_random.c.o
[ 25%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_random_f.c.o
[ 25%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_strat.c.o
[ 25%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_version.c.o
[ 25%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/library_version_f.c.o
[ 25%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/mapping.c.o
[ 25%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/mapping_io.c.o
[ 25%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/mesh.c.o
[ 25%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/mesh_check.c.o
[ 26%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/mesh_coarsen.c.o
[ 26%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/mesh_graph.c.o
[ 26%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/mesh_induce_sepa.c.o
[ 26%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/mesh_io.c.o
[ 26%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/mesh_io_habo.c.o
[ 26%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/mesh_io_scot.c.o
[ 26%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/order.c.o
[ 26%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/order_check.c.o
[ 27%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/order_io.c.o
[ 27%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/parser.c.o
[ 27%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/parser_ll.c.o
[ 27%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/parser_yy.c.o
[ 27%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vgraph.c.o
[ 27%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vgraph_check.c.o
[ 27%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vgraph_separate_bd.c.o
[ 27%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vgraph_separate_df.c.o
[ 27%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vgraph_separate_es.c.o
[ 28%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vgraph_separate_fm.c.o
[ 28%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vgraph_separate_gg.c.o
[ 28%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vgraph_separate_gp.c.o
[ 28%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vgraph_separate_ml.c.o
[ 28%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vgraph_separate_st.c.o
[ 28%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vgraph_separate_th.c.o
[ 28%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vgraph_separate_vw.c.o
[ 28%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vgraph_separate_zr.c.o
[ 29%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vgraph_store.c.o
[ 29%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vmesh.c.o
[ 29%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vmesh_check.c.o
[ 29%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vmesh_separate_fm.c.o
/Users/samsonadler/neper/src/contrib/scotch/src/libscotch/vmesh_separate_fm.c:359:57: warning: for loop has empty body [-Wempty-body]
  for (hashmax = 256; hashmax < hashnbr; hashmax <<= 1) ; /* Get upper power of two */
                                                        ^
/Users/samsonadler/neper/src/contrib/scotch/src/libscotch/vmesh_separate_fm.c:359:57: note: put the semicolon on a separate line to silence this warning
1 warning generated.
[ 29%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vmesh_separate_gg.c.o
[ 29%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vmesh_separate_gr.c.o
[ 29%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vmesh_separate_ml.c.o
[ 29%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vmesh_separate_zr.c.o
[ 30%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vmesh_separate_st.c.o
[ 30%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/vmesh_store.c.o
[ 30%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/wgraph.c.o
[ 30%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/wgraph_check.c.o
[ 30%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/wgraph_part_es.c.o
[ 30%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/wgraph_part_fm.c.o
[ 30%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/wgraph_part_gg.c.o
[ 30%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/wgraph_part_ml.c.o
[ 31%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/wgraph_part_rb.c.o
[ 31%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/wgraph_part_st.c.o
[ 31%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/wgraph_part_zr.c.o
[ 31%] Building C object contrib/scotch/CMakeFiles/scotch.dir/src/libscotch/wgraph_store.c.o
[ 31%] Linking C shared library libscotch.dylib
Undefined symbols for architecture arm64:
  "_SCOTCH_errorPrint", referenced from:
      _archLoad in arch.c.o
      _archSave in arch.c.o
      _archDecoArchBuild in arch_build.c.o
      _archDeco2ArchBuild in arch_build2.c.o
      _archDeco2BuildMatchMate in arch_build2.c.o
      _archCmpltArchLoad in arch_cmplt.c.o
      _archCmpltArchSave in arch_cmplt.c.o
      ...
  "_SCOTCH_errorPrintW", referenced from:
      _stratParserParse2 in parser_yy.c.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [contrib/scotch/libscotch.dylib] Error 1
make[1]: *** [contrib/scotch/CMakeFiles/scotch.dir/all] Error 2
make: *** [all] Error 2

Does anyone know how to fix this? Can I even install Neper on an M1 Mac? Any help would be appreciated. Thanks!

**Comment by rquey** [comment]
I have very little experience with Mac and absolutely no experience with Mac M1, sorry.

If someone does, please help.

**Comment by xshang93** [comment]
This looks a lot like the issue I had before. The only time I made neper work on an M1 mac was when I backed up everything from an intel mac, where neper can be successfully installed, and then restore everything onto an M1 mac. Otherwise I wasn't able to find a solution.
